backend/generated/schema.graphql

```diff
@@ -147,24 +147,24 @@ type bridgeTransactionPage {
 }
 
 type bridgeTransaction {
-  amount: BigInt!
```

Comment:
Since we sometimes create these records from the L1, we don't have that value and the ones below.